frontend/src/modules/machinery/components/source/ConcordanceSearch.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+type Props = {|
+    projectName?: string,
+|};
+
+/**
+ * Show the translation source from Pontoon's memory.
+ */
+export default function TranslationMemory(props: Props) {
```

Comment:
We should change the name of the function.

Maybe also amend the comment a little to differentiate from `TranslationMemory`,